docs/content/modules/player-lifecycle-feature-module.md

```diff
@@ -14,32 +14,40 @@ Before reading this document, make sure you are familiar with:
 
 The Player lifecycle module is a feature module providing you with a simple player spawning and player lifecycle management implementation.
 
-### How to enable this module
+## How to enable this module
 
 To enable this module, you need to add the necessary systems to your workers in the `HandleWorkerConnectionEstablished()` of your [`WorkerConnector`]({{urlRoot}}/content/gameobject/creating-workers-with-workerconnector).
 After your worker has been created, you need to use the following code snippet:
 
 1. On a [client-worker]({{urlRoot}}/content/glossary#client-worker): `PlayerLifecycleHelper.AddClientSystems(Worker.World)`
+  * By default, the module automatically sends a player creation request as soon as the client worker connects to SpatialOS. To manually initiate player creation or provide arbitrary data in a player creation request, add an extra `false` argument when adding client systems: `PlayerLifecycleHelper.AddClientSystems(Worker.World, false);`
 1. On a [server-worker]({{urlRoot}}/content/glossary#server-worker): `PlayerLifecycleHelper.AddServerSystems(Worker.World)`
 
-### How to spawn a player entity
+The Player lifecycle module provides a `PlayerLifecycleConfig` configuration class, containing a field called `AutoRequestPlayerCreation`. By default this is set to true, in which case the client-worker initiates the player entity creation as soon as it has connected to SpatialOS.
+
+Calling `PlayerLifecycleHelper.AddClientSystems(Worker.World, false)` sets this value to false, in which case you must manually initiate player creation.
```

Comment:
Why is this configuration info here and is it strictly necessary? 

As a user I just need to know that if I want to manually send the request, to set the optional argument to false.